test/e2e/docker/Dockerfile

```diff
@@ -4,10 +4,19 @@
 FROM golang:1.18
 
 RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
-RUN apt-get -qq install -y libleveldb-dev librocksdb-dev make libc-dev libtool >/dev/null
+RUN apt-get -qq install -y libleveldb-dev make libc-dev libtool >/dev/null
 
 ARG SRCDIR=/src/ostracon
 
+ARG ROCKSDB_VERSION=6.29.5
```

Comment:
By checking my local, I could make build success with `tendermint/tm-db-v0.6.7` and `facebook/rocksdb` `v6.24.2` to `v6.29.5`(cannot build the success with `v7.0.1+`).  Would you leave a comment on this above with the supported versions(`tm-db` and `rocksdb`)? 

And would you leave a comment about the version of `apt get librocksdb-dev` will be installed `6.11.4` is older than `v6.24.2`.

Reply:
Oh, yes. I've confirmed I can compile with RocksDB v6.24.2 and am trying to confirm with v6.23.3. The functions that caused build errors were seems to be added with this https://github.com/facebook/rocksdb/commit/d4b75d295fd7c18976ff652ed04b3df3624a53f6 commit, and it seems to be merged into `main` branch at https://github.com/facebook/rocksdb/pull/8526 .

btw, is the comment meant for this PR? or in the Dockerfile?

Reply:
> this above

Please leave it in the `Dokcerfile`.

Reply:
Added a comment on the versions and source build.